pages/history.js

```diff
@@ -39,12 +40,23 @@ export default function HistoryPage() {
 				<title>History - The Snakeroom</title>
 			</Head>
 			<EventsContainer>
+				<Event icon={faPalette} name="Place" time="April 2022">
+					For the second iteration of Place, players were invited to
+					place a pixel on a blank canvas once every 5 minutes. Each
+					day of the event another 1000x1000 pixel portion of the
+					canvas was made available. Sneknet was developed in two
```

Comment:
Only the first canvas expansion was 1000 by 1000 pixels. The second canvas expansion was 2000 by 1000 pixels.